spec.html

```diff
@@ -31177,7 +31177,7 @@ <h1>Number.prototype.toExponential ( _fractionDigits_ )</h1>
             1. If _fractionDigits_ is not *undefined*, then
               1. Let _e_ and _n_ be integers such that 10<sup>_f_</sup> ≤ _n_ &lt; 10<sup>_f_ + 1</sup> and for which _n_ × 10<sup>_e_ - _f_</sup> - _x_ is as close to zero as possible. If there are two such sets of _e_ and _n_, pick the _e_ and _n_ for which _n_ × 10<sup>_e_ - _f_</sup> is larger.
             1. Else,
-              1. [id="step-number-proto-toexponential-intermediate-values"] Let _e_, _n_, and _f_ be integers such that _f_ ≥ 0, 10<sup>_f_</sup> ≤ _n_ &lt; 10<sup>_f_ + 1</sup>, 𝔽(_n_ × 10<sup>_e_ - _f_</sup>) is 𝔽(_x_), and _f_ is as small as possible. Note that the decimal representation of _n_ has _f_ + 1 digits, _n_ is not divisible by 10, and the least significant digit of _n_ is not necessarily uniquely determined by these criteria.
+              1. [id="step-number-proto-toexponential-intermediate-values"] Let _e_, _n_, and _ff_ be integers such that _ff_ ≥ 0, 10<sup>_ff_</sup> ≤ _n_ &lt; 10<sup>_ff_ + 1</sup>, 𝔽(_n_ × 10<sup>_e_ - _ff_</sup>) is 𝔽(_x_), and _ff_ is as small as possible. Note that the decimal representation of _n_ has _ff_ + 1 digits, _n_ is not divisible by 10, and the least significant digit of _n_ is not necessarily uniquely determined by these criteria.
```

Comment:
The outer `f` is used in the other branch in exactly the same what that `ff` is used here, so I think that would be more confusing.

Reply:
I don't see why that makes it confusing.

Reply:
Because it suggests a dissimilarity between the branches where none exists, unless I'm misunderstanding your suggestion.

Reply:
I don't think you can just rename `_f_` in this one step, because there are subsequent reference to `_f_` which might or might not be *this* `_f_`.

So you can make this change, but you'd need to add a step `Set _f_ to _ff_.`

(From https://github.com/tc39/ecma262/pull/3149#discussion_r1299217210)

Reply:
Fixed.